gdcdictionary/schemas/_terms.yaml

```diff
@@ -64,54 +287,143 @@ base_caller_version: # TOREVIEW
+    The average number of cigarettes smoked per day.
   termDef:
     term: Smoking Use Average Number
     source: caDSR
     cde_id: 2001716
     cde_version: 4.0
     term_url: "https://cdebrowser.nci.nih.gov/CDEBrowser/search?elementDetails=9&FirstTimer=0&PageId=ElementDetailsGroup&publicId=2001716&version=4.0"
 
+circumferential_resection_margin:
+  description: >
+    A value in millimeters indicating the measured length between a malignant lesion of the
+    colon or rectum and the nearest radial (or circumferential) border of tissue removed during
+    cancer surgery.
+  termDef:
+    term: Colorectal Surgical Margin Circumferential Distance Measurement
+    source: caDSR
+    cde_id: 64202
+    cde_version: 3.0
+    term_url: "https://cdebrowser.nci.nih.gov/CDEBrowser/search?elementDetails=9&FirstTimer=0&PageId=ElementDetailsGroup&publicId=64202&version=3.0"
+
 classification_of_tumor:
   description: >
-    Text that describes the kind of disease present in the tumor specimen as related
-    to a specific timepoint.  
+    Text that describes the kind of disease present in the tumor specimen as related to a specific
+    timepoint.
   termDef:
-    term: Tumor Type 
-    source: caBIG
+    term: Tumor Tissue Disease Description Type
+    source: caDSR
     cde_id: 3288124
     cde_version: 1.0
     term_url: "https://cdebrowser.nci.nih.gov/CDEBrowser/search?elementDetails=9&FirstTimer=0&PageId=ElementDetailsGroup&publicId=3288124&version=1.0"
 
-concentration: # TOREVIEW
+colon_polyps_history:
+  description: >
+    Yes/No indicator to describe if the subject had a previous history of colon polyps as noted
+    in the history/physical or previous endoscopic report (s).
+  termDef:
+    term: Colon Carcinoma Polyp Occurrence Indicator
+    source: caDSR
+    cde_id: 3107197
+    cde_version: 1.0
+    term_url: "https://cdebrowser.nci.nih.gov/CDEBrowser/search?elementDetails=9&FirstTimer=0&PageId=ElementDetailsGroup&publicId=3107197&version=1.0"
+
+composition:
+  description: >
+    Text term that represents the cellular composition of the sample.
+  termDef:
+    term: Biospecimen Cellular Composition Type
+    source: caDSR
+    cde_id: 5432591
+    cde_version: 1.0
+    term_url: "https://cdebrowser.nci.nih.gov/CDEBrowser/search?elementDetails=9&FirstTimer=0&PageId=ElementDetailsGroup&publicId=5432591&version=1.0"
+
+concentration:
   description: >
-    Concentration of the item, in molarity. 
+    Numeric value that represents the concentration of an analyte or aliquot extracted from the
+    sample or sample portion, measured in milligrams per milliliter.
+  termDef:
+    term: Biospecimen Analyte or Aliquot Extracted Concentration Milligram per Milliliter Value
+    source: caDSR
+    cde_id: 5432594
+    cde_version: 1.0
+    term_url: "https://cdebrowser.nci.nih.gov/CDEBrowser/search?elementDetails=9&FirstTimer=0&PageId=ElementDetailsGroup&publicId=5432594&version=1.0"
 
-current_weight: # TOREVIEW
+creation_datetime:
```

Comment:
Ah I missed that it was already in the dictionary in the portion node - this is OK then.